Solutions/Marain.Operations.Hosting.AspNetCore/Microsoft/Extensions/DependencyInjection/OperationsRepositoryServiceCollectionExtensions.cs

```diff
@@ -49,8 +47,8 @@ public static class OperationsRepositoryServiceCollectionExtensions
             this IServiceCollection services,
             Action<IOpenApiHostConfiguration>? configureHost = null)
         {
-            services.AddTenancyBlobContainerOperationsRepository();
             services.AddOperationsStatusApi(configureHost);
+            services.AddTenancyBlobContainerOperationsRepository();
```

Comment:
Is there a reason you changed the order here? If so, I'd like a comment because I have no idea why this is necessary, and it's the sort of thing that could be messed up later because nobody realised order was significant.

Reply:
Comment added.